src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -201,22 +201,20 @@ private void visitLiteralFor(DetailAST ast) {
      * @param ast the token to check.
      */
     private void processExpression(DetailAST ast) {
-        if (ast.branchContains(TokenTypes.LPAREN)) {
```

Comment:
This was surviving and regression show no way to keep it so I am removing it. We are trying to remove `branchContains` anyways, so I see no reason to try and look into it more.